python/cugraph/tests/dask/opg_pagerank_testing.py

```diff
@@ -38,10 +35,28 @@ def test_dask_pagerank():
                              names=['src', 'dst', 'value'],
                              dtype=['int32', 'int32', 'float32'])
 
-    g = cugraph.DiGraph()
-    g.from_dask_cudf_edgelist(ddf)
+    df = cudf.read_csv(input_data_path,
+                       delimiter=' ',
+                       names=['src', 'dst', 'value'],
+                       dtype=['int32', 'int32', 'float32'])
 
-    dcg.pagerank(g)
+    g = cugraph.DiGraph()
+    g.from_cudf_edgelist(df, 'src', 'dst')
+
+    dg = cugraph.DiGraph()
+    dg.from_dask_cudf_edgelist(ddf)
```

Comment:
1. How do we support datasets stored in multiple files?
2. If the file is on an NFS this will be limited by network speed. I wonder if each node first gets a local copy before parsing with the GPU.

Reply:
dask read_csv takes care of reading from multiple files, just like snmg version.